backend/src/main/java/com/swe573/socialhub/controller/ServiceController.java

```diff
@@ -61,7 +61,7 @@ public ResponseEntity<ServiceDto> findServiceById(@PathVariable(value = "id") lo
         }
     }
 
-    @DeleteMapping("/{id}")
+    @PostMapping("/delete/{id}")
```

Comment:
we've decided to use deletes instead of posts for backend, no?

Reply:
yes

Reply:
pinged deniz about this in case she didn't see here.

Reply:
sorry for the delay. fixed it.